docs/guides/modules/permissions-authentication/pages/mfa.adoc

```diff
@@ -22,65 +19,105 @@ To configure MFA on your account, follow these steps:
 
-It is also possible to intentionally regenerate your MFA recovery code. To do this, follow these steps:
+To intentionally regenerate your MFA recovery code:
 
 . In the link:https://app.circleci.com/home/[CircleCI web app], select your profile from the upper right corner, then select **User Settings**.
 . Select **Password & authentication** from the sidebar.
-. On the Password & authentication page, in the **Multi-factor authentication**section, select **Add/edit authenticator app**.
-. Input your password at the password prompt.
+. In the "Multi-factor authentication" section, select btn:[Add/edit authenticator app].
+. Input your password at the password prompt. This is the password you use to sign in to CircleCI.
 . Input your OTP at the OTP prompt. Once these have been successfully submitted, your OTP ID and recovery code ID will be displayed.
 . Select the button to **Regenerate recovery code** on the recovery code row.
-. At the prompt to regenerate the recovery code, select `Yes, regenerate code`. This will generate a new code and invalidate the previous recovery code. Store the new code somewhere safe. It will not be displayed again.
+. At the prompt to regenerate the recovery code, select btn:[Yes, regenerate code]. This will generate a new recovery code and invalidate the previous one. Store the new code in a safe place. It will not be displayed again.
 
 image::guides:ROOT:authentication/mfa-regenerate-recovery-code.png[Password and authentication page with regenerate recovery code button]
 
 image::guides:ROOT:authentication/mfa-display-recovery-code.png[Password and authentication page displaying new recovery code]
 
+[#mfa-reset-process]
+== MFA Reset Process
+
+The MFA reset process depends on your situation and whether you retain access to your account.
+
+=== Scenario 1: You still have account access
+
+If you can log in and want to reconfigure your authenticator app or regenerate your recovery code:
+
+. Go to **User Settings** in the CircleCI web app.
+. Select **Password & authentication**.
+. In the **Multi-factor authentication** section, select **Add/edit authenticator app**.
+. Enter your password and OTP when prompted.
+. Follow the setup steps to scan the new QR code with your authenticator app.
+. Click **Regenerate recovery code** to generate a new code.
+. **Store the new recovery code securely** – it will only be displayed once, and the previous code becomes invalid.
+
+=== Scenario 2: You are locked out (no authenticator app or recovery code)
+
+If you have lost access to both your authenticator app and recovery code, you are locked out of your CircleCI account. CircleCI cannot bypass MFA for security reasons. The only way to regain access is through your recovery code. Support cannot reset MFA if you have no recovery code and cannot verify your identity.
+
+=== Scenario 3: Support-initiated MFA reset
```

Comment:
Does this 3rd scenario mean that if you lose your recovery code you might actually be able to regain access with help from support? As this contradicts the previous section we should maybe tighten up the language a bit?